web/src/components/ak-event-info.ts

```diff
@@ -0,0 +1,484 @@
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import { VERSION } from "@goauthentik/common/constants";
+import { EventContext, EventModel, EventWithContext } from "@goauthentik/common/events";
+import { AKElement } from "@goauthentik/elements/Base";
+import "@goauthentik/elements/Expand";
+import "@goauthentik/elements/Spinner";
+import { PFSize } from "@goauthentik/elements/Spinner";
+
+import { msg, str } from "@lit/localize";
+import { CSSResult, TemplateResult, css, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { map } from "lit/directives/map.js";
+import { until } from "lit/directives/until.js";
+
+import PFButton from "@patternfly/patternfly/components/Button/button.css";
+import PFCard from "@patternfly/patternfly/components/Card/card.css";
+import PFDescriptionList from "@patternfly/patternfly/components/DescriptionList/description-list.css";
+import PFList from "@patternfly/patternfly/components/List/list.css";
+import PFFlex from "@patternfly/patternfly/layouts/Flex/flex.css";
+import PFBase from "@patternfly/patternfly/patternfly-base.css";
+
+import { EventActions, FlowsApi } from "@goauthentik/api";
+
+type Pair = [string, string | number | EventContext | EventModel | string[] | TemplateResult];
+
+// https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/about-automation-for-issues-and-pull-requests-with-query-parameters
+
+const emailMessageBody = (context: EventContext) => `
```

Comment:
small nit: this is not for an email, this is a template that is passed to github via querystring (which is kind of a hacky solution), but it provides a link to open a github issue that includes the full stacktrace